templates/bionomia.md

```diff
@@ -1,20 +1,28 @@
 ---
 {{#if orcid}}ORCID: {{orcid}}{{/if}}
 {{#if wikidata}}Wikidata: {{wikidata}}{{/if}}
-aliases: ["{{{query}}}"]
+uri: {{uri}}
```

Comment:
Please can I just check that `other_names` includes the search string passed to the search service? That was the content of the `query` variable, allowing aliases to more easily propose links.

Reply:
On consideration I don't think bionomia should include the query string in its `other_names`: it is the responsibility of this template to include it (and maybe should be a configurable setting in the script options)